emails/utils.py

```diff
@@ -85,7 +85,7 @@ def get_trackers(level):
 
 def download_trackers(repo_url, category="Email"):
     # email tracker lists from shavar-prod-list as per agreed use under license:
-    resp = requests.get(repo_url)
+    resp = requests.get(repo_url, timeout=10)
```

Comment:
this one does seem safer, but @say-yawn may know if it is slow

Reply:
I agree that 10 seconds is enough for us to get response back from GitHub for the tracker list. This request also only gets called during CircleCI jobs so it should be safe for the response time to take longer.